app/models/katello/content_view_version_import_history.rb

```diff
@@ -0,0 +1,26 @@
+module Katello
+  class ContentViewVersionImportHistory < Katello::Model
```

Comment:
Wonder if should have an api end point to facilitate searching.
```hammer content-import list``` is a good goal imo. I am open to addressing that in a different PR, but if its not too hard I 'd recommend putting it here ?

Reply:
Agree, I don't think it'd be good to have `hammer content-export list` but no `hammer content-import list`.  I'll start working on it; should probably be small enough to fit in this PR.